### PR TITLE
Nft tutorial update

### PR DESCRIPTION
**Pull request by bxpana**
Update NFT tutorials to include steps on minting a metadata.json file so users have a preview of their NFT.

### CODE REVIEW

File: docs/faq/tutorials.md [hunk 1]
```diff
@@ -251,8 +251,9 @@ A single yellow check mark next to your balance indicates the zkSync server has
 
 ## Mint an NFT
 
-The following steps will cover using IPFS desktop to mint an NFT. **This process will mint an NFT of your image location on IPFS and will not be available for preview if withdrawn to L1.** 
-To mint an NFT with an IPFS location and a preview of the image, consider using one of the community-created frontends&ast; like [zkNFT](https://zknft.xyz/#/) or [Open Sky](https://open-sky.vercel.app/).
```
Comment (barakshani) [inline docs/faq/tutorials.md:255]: what's the reason for omitting these parts?

File: docs/faq/tutorials.md [hunk 2]
```diff
@@ -291,7 +326,7 @@ To mint an NFT with an IPFS location and a preview of the image, consider using
 > - Signing the message will generate your zkSync Layer 2 private key (remember, it is good practice to read the messages you sign).
 </br>
 
-5. Enter the copied CID (Content Identifier) in "Content Address" and click "Mint NFT."
+9. Enter the copied CID (Content Identifier) of your .json file in "Content Address" and click "Mint NFT."
```
Comment (barakshani) [inline docs/faq/tutorials.md]: Let's have "(Content Identifier)" at the first appearance of CID in the text. I see one at point 5 (but maybe there's an earlier one)

File: docs/faq/tutorials.md (hunk 1 above)
Comment (bxpana) [reply docs/faq/tutorials.md:255]: The new directions will allow users to have an image if they withdraw to L1

**Review by barakshani — APPROVED** [review]

**Pull request merged**